lib/ain-rs-exports/src/evm.rs

```diff
@@ -592,6 +592,27 @@ pub fn evm_try_get_block_number_by_hash(result: &mut ffi::CrossBoundaryResult, h
     }
 }
 
+pub fn evm_try_get_dvm_block_number_by_number(
+    result: &mut ffi::CrossBoundaryResult,
+    number: u64,
+) -> u64 {
+    match SERVICES.evm.storage.get_block_by_number(&U256::from(number)) {
+        Ok(Some(block)) => {
+            let extra_data = block.header.extra_data;
+            let data = String::from_utf8(extra_data.to_vec()).unwrap();
```

Comment:
the only dvm info for evm 
ref: https://github.com/DeFiCh/ain/blob/9bfd6bb665340674f793614341fed1d1ee276b35/lib/ain-evm/src/evm.rs#L341